core/client_test.go

```diff
@@ -10,20 +10,20 @@ import (
 )
 
 func TestRemoteClient_Status(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*3)
```

Comment:
I increased the timeout because I saw  random timeouts when running tests locally